src/common/Placeholder.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { css, keyframes } from '@emotion/react';
+
+import { space, neutral } from '@guardian/source-foundations';
+
+const BACKGROUND_COLOUR = neutral[93];
+
+type Props = {
+    height: number;
+    rootId?: string;
+    width?: number;
+    spaceBelow?: 1 | 2 | 3 | 4 | 5 | 6 | 9;
+    spaceLeft?: 1 | 2 | 3 | 4 | 5 | 6 | 9;
+    shouldShimmer?: boolean;
+};
```

Comment:
Actually, a small suggestion to keep things DRY.

```suggestion

type Props = {
    height: number;
    rootId?: string;
    width?: number;
    spaceBelow?: keyof typeof space;
    spaceLeft?: typeof typeof space;
    shouldShimmer?: boolean;
};
```

Reply:
I ended up removing these props because they're actually never used and it doesn't make sense to have this component be responsible for the margins around it in any case